ofono/modem.go

```diff
@@ -390,7 +469,7 @@ func (modem *Modem) GetMMSContexts(preferredContext dbus.ObjectPath) (mmsContext
 	}
 
 	for _, context := range contexts {
-		if (context.isTypeInternet() && context.isActive() && context.hasMessageCenter()) || context.isTypeMMS() {
+		if (context.isTypeInternet() && context.hasMessageCenter()) || context.isTypeMMS() {
```

Comment:
In this concrete case I think the parenthesis add readability so I would prefer to keep them.